spec/relativetimeformat.html

```diff
@@ -79,12 +72,11 @@
         1. If _exists_ is *false*, then
           1. Let _entry_ be _unit_.
         1. Let _patterns_ be ? Get(_fields_, _entry_).
-        1. Let _v_ be _units_.[[&lt;_unit_&gt;]].
-        1. If _v_ is between -2 (inclusive) and 2 (inclusive), then
-          1. Let _exists_ be ? HasProperty(_patterns_, ToString(_v_)).
-          1. If _exists_ is *true*, then
-            1. Let _result_ be Get(_patterns_, ToString(_v_)).
-            1. Return _result_.
+        1. Let _v_ be _value_.[[&lt;_unit_&gt;]].
```

Comment:
going forward we want to avoid accessing internal slots for computed values, e.g.: `[[<x>]]`, I'm ok with this for now.